components/dashboard/donation/editDonationModalBody.js

```diff
@@ -63,19 +62,12 @@ const EditDonationModalBody = ({ donationAmount, isNewDonor, onClose }) => {
       return
     }
 
-    let token
-    try {
-      const cardElement = elements.getElement(CardElement)
-      const res = await stripe.createToken(cardElement)
-      token = res.token
-    } catch (e) {
-      setSubmitError(e.message)
-      return
-    }
+    const cardElement = elements.getElement(CardElement)
+    const res = await stripe.createToken(cardElement)
+    const token = res.token
```

Comment:
removed this from the try catch block cause i want the error to throw into the catch block of the "handleSaveChanges" method